validator-api/validator_api/config.py

```diff
@@ -92,6 +92,7 @@ def parse_proxies(proxy_list: List[str]) -> List[str]:
 
 MAX_FOCUS_POINTS_PER_HOUR = int(os.getenv("MAX_FOCUS_POINTS_PER_HOUR", 80))  # $80 / hour
 FIXED_TAO_USD_ESTIMATE = float(os.getenv("FIXED_TAO_USD_ESTIMATE", 300.0))
+BOOSTED_TASKS_PERCENTAGE = float(os.getenv("BOOSTED_TASKS_PERCENTAGE", 0.7))
```

Comment:
maybe rename to OMEGA_TASKS_REWARD_ALLOC or something like that? because they may not necessarily be "boosted" in the future / provide extra reward